server/package.json

```diff
@@ -13,7 +13,9 @@
     "express": "~4.16.1",
     "http-errors": "~1.6.3",
     "morgan": "~1.9.1",
-    "pug": "2.0.0-beta11"
+    "pug": "2.0.0-beta11",
+    "react": "^18.2.0",
```

Comment:
Server shouldn't depend on `react`, do you remember what caused this to get added to the package.json?

Reply:
@MitchellCWM added these here: https://github.com/ivysaur-unb/on-call-coordinator/commit/1ed35bdbd82146561b5f484ebd4e1ca9e877031a
It doesn't seem used anywhere, maybe a local change that got reverted before committing. Should be safe to remove these then run `npm install` to update `package-lock.json`

Reply:
removed & re-ran npm install, also added in nodemon in the process.